lib/giphy/version.rb

```diff
@@ -1,3 +1,3 @@
 module Giphy
-  VERSION = "3.0.0"
+  VERSION = "3.0.1"
```

Comment:
```suggestion
  VERSION = "3.1.0"
```
looks more appropriate as it extends existing gem API and it's compatible with previous versions